### PR TITLE
add jira badge to the readme

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

**Comment by pavolloffay** [comment]
@objectiser could you please review?

File: README.adoc [hunk 1]
```diff
@@ -3,7 +3,9 @@
 
 ifdef::env-github[]
 [link=https://travis-ci.org/hawkular/hawkular-apm]
-image::https://travis-ci.org/hawkular/hawkular-apm.svg?branch=master[Build Status,70,18]
+image::https://travis-ci.org/hawkular/hawkular-apm.svg?branch=master[Build Status]
+[link=https://issues.jboss.org/browse/HWKAPM-602?filter=12328180]
```
Comment (objectiser) [inline README.adoc]: why is it linking to a specific issue?

Comment (pavolloffay) [inline README.adoc]: Jira always adds a specific issue (the first one) to that link. However, it is good suggestion link does not work in incognito mode.

This link should work:
https://issues.jboss.org/issues?jql=project%20%3D%20HWKAPM%20AND%20status%20%3D%20Open

**Comment by objectiser** [comment]
Unfortunately that link isn't project specific - could you use "https://issues.jboss.org/projects/HWKAPM/issues" instead?

**Comment by pavolloffay** [comment]
When I closed incognito previous link did not work, Jira probably sets something to cookies..

@objectiser thanks.

**Comment by objectiser** [comment]
@pavolloffay LGTM :+1:

**Pull request merged**